### DISCUSSION TITLE
Is It possible to add Mon-Sun labels within the Dates?

### DISCUSSION THREAD

**Discussion by amarg26** [dmuy/duDatepicker#31] [Q&A] [answered]
@dmuy Thank you so much for implementing my previous feature suggestion in new DuDatepicker. 🤝 

I am reusing your DuDatepicker for the current requirements as given below URL.

[#Question, #Request. Can we customize as per we want ?](https://github.com/dubrox/Multiple-Dates-Picker-for-jQuery-UI/issues/263)

I initially used Jquery-MutidatePicker, It works exactly like this Datepicker but customizing it quite tough and challenging. ❌ 

Using Current DuDatepicker I am able to achieve almost everything. 

I just need 2 things as below.

1. Display DuDatepicker by default instead of input click or focus. Way to display it by default.

2. Display Mon-Sun respective to their dates within the date cells dynamically. (You can refer the URL)

Please let me know if you need more information.

Thanks in Advance. 👍

**Comment by dmuy** [comment, marked as answer]
Hi @amarg26,

As far as I understood your requirements for a date picker, I don't think this library would be ideal for that. The following are the things I have considered:

1. There's so little space on the date cells to add any more text on it, and it would just clutter up the picker UI.
2. The library's intended use is a date picker (for input elements - hidden or not) which is why users should click on something before showing it, and the picker being visible by default falls into the category of a calendar (for me). As much as I would like to help you out, I don't want the library to have functionalities which (I think) would deviate from it's original intended use.
3. You might want to reconsider how your application UI displays the selected date(s) and not embed the date picker (this library) itself.
4. If I wanted to accommodate such functionalities, I would rather create another library for this kind of requirement.

I hope I understood your requests correctly, and I hope you understand my stand on it.

**Reply by amarg26** [reply]
Hi @dmuy 

Thanks for the quick reply.

-> 1. For this I can manage using CSS for sure. As I already created static dummy of it.
-> 2. I understand your point of you completely, As of now somehow I want to display current month calendar by default without any input click open. Then Mon-Sun labels within Dates.

 It will be a big favor. Please let me know whether it can be possible. 🙏
